### PR TITLE
fix: initialize EME for all playlists and PSSH values

### PR DESCRIPTION
**Pull request by gesinger**
## Requirements Checklist
- [X] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [X] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](https://jsbin.com/gejugat/edit?html,output))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: src/videojs-http-streaming.js [hunk 1]
```diff
@@ -172,11 +172,75 @@ const emeKeySystems = (keySystemOptions, videoPlaylist, audioPlaylist) => {
...
+ * @param {Object[]} mainPlaylists
+ *        The playlists found on the master playlist object
+ */
 const setupEmeOptions = ({
   player,
   sourceKeySystems,
   media,
-  audioMedia
+  audioMedia,
+  mainPlaylists
```
Comment (gkatsev) [inline src/videojs-http-streaming.js:243]: Seems like this is a newly required attribute? Can we default it to an empty array and not break if this method is called without it?

Comment (gesinger) [reply src/videojs-http-streaming.js:243]: It shouldn't be called anywhere except from the one place we call it internally, which will always provide it.

Comment (gesinger) [reply src/videojs-http-streaming.js:243]: And from tests.

Comment (gkatsev) [reply src/videojs-http-streaming.js:243]: I thought that setupEmeOptions already existed in a released version but it doesn't, so, it doesn't matter.

**Pull request merged**